pyedflib/edfwriter.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, file_name, n_channels,
         self.recording_start_time = datetime.now()
         self.birthdate = date(1900, 1, 1)
         self.duration = 1
+        self.number_of_annotations = 1
```

Comment:
annotations are only 1 for BDFplus and EDFplus.
For BDF and EDF, annotations ore 0.

Reply:
Please add
if self.file_type == FILETYPE_EDFPLUS or self.file_type == FILETYPE_BDFPLUS:
self.number_of_annotations = 1
else:
self.number_of_annotations = 0